debian/control

```diff
@@ -31,6 +31,7 @@ Build-Depends: debhelper (>= 9.20141010),
                libselinux1-dev (>= 2.1.9),
                libacl1-dev,
                liblzma-dev,
+               liblz4-dev (>= 0.0~r125),
```

Comment:
Why is this being added here? It doesn't seem to have anything to do with networkd.